dist/common/scripts/scylla_coredump_setup

```diff
@@ -62,13 +65,35 @@ ExternalSizeMax=1024G
         with open('/etc/systemd/coredump.conf', 'w') as f:
             conf = f.write(conf_data)
         if args.dump_to_raiddir:
-            rmtree('/var/lib/systemd/coredump')
+            with open('/etc/fstab') as f:
+                fstab = f.readlines()
+            for f in fstab:
+                if re.match(r'^/var/lib/scylla/coredump', f):
+                    print('/etc/fstab already has bind-mount entry', file=sys.stderr)
+                    sys.exit(1)
             makedirs('/var/lib/scylla/coredump')
-            os.symlink('/var/lib/scylla/coredump', '/var/lib/systemd/coredump')
+            with open('/etc/fstab', 'a') as f:
+                f.write('/var/lib/scylla/coredump /var/lib/systemd/coredump none defaults,bind 0 0\n')
+            run('mount /var/lib/systemd/coredump')
```

Comment:
It's safer to use a [systemd.mount](https://www.freedesktop.org/software/systemd/man/systemd.mount.html) file, we don't risk corrupting /etc/fstab. We can also set Before dependencies so it is set up before scylla is launched.

Reply:
Updated the code using systemd.mount.